ferveo/benches/benchmarks/pvdkg.rs

```diff
@@ -66,16 +63,16 @@ pub fn setup_dkg(
 ) -> PubliclyVerifiableDkg<EllipticCurve> {
     let keypairs = gen_keypairs(num);
     let validators = gen_validators(&keypairs);
-    let me = validators.validators[validator].clone();
+    let me = validators[validator].clone();
     PubliclyVerifiableDkg::new(
         validators,
         Params {
             tau: 0,
             security_threshold: 300 / 3,
```

Comment:
where is 300 coming from?

Reply:
It's `shares_num`, from the line below. Should be reused here.